environment.yaml

```diff
@@ -0,0 +1,13 @@
+name: griddataformats-docs
```

Comment:
This should just be merged into what's in `ci/environment.yaml`

Reply:
I know. Just figured it out locally.